packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -316,6 +342,145 @@ export class SharedTree
 	}
 }
 
+/**
+ * Get a {@link BranchableTree} from a {@link ITree}.
+ * @remarks The branch can be used for "version control"-style coordination of edits on the tree.
+ * @alpha
+ */
+export function getBranch(tree: ITree): BranchableTree;
```

Comment:
Nit: "get" here implies to me that you're accessing something that already may exist. Would "create" make more sense?

Reply:
I'm afraid that `create` sounds a lot like it is "forking/branching", which it's not. This function is nothing but a hack to work around the fact that we can't add @alpha methods to SharedTree (see other comment) and so we have to have a way to "cast the public ITree which has no branching capabilities to the alpha BranchableTree which does have alpha capabilities". So conceptually, it's not actually creating anything, it's just giving you "the (already existing) branch that corresponds to the object you provided. 

This is my first attempt at trying to introduce an alpha API, and the fact that we can't mark individual methods/properties as @alpha really cripples our options. I'm not sure what else to do but I'm super open to more ideas, if you have some (or, a better name - maybe just not "create").

Reply:
Ooh, okay. I misunderstood. Might be worth expanding the documentation a bit to clarify the situation a bit more. FWIW, I think it's okay to call out that this is exposed the way it is for now based on its experimental status, but the long term direction will be different. That's useful info for anyone wanting to leverage experimental APIs.

Reply:
I added a disclaimer to make it obvious that the API might change, and I included a note about `getBranch` being removed eventually (though, as private remarks, not public). Let me know if you have more ideas.